docs：Add Biweekly-Report (#1633)
iresty/iresty.github.io: blog/en/blog/2023/07/05/weekly-report.md

## Contributor Statistics

![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2023/07/03/NmhS4JKj_%E8%B4%A1%E7%8C%AE%E8%80%85%E5%90%8D%E5%8D%95.png)

![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2023/07/03/KCxDtAOZ_%E6%96%B0%E6%99%8B%E8%B4%A1%E7%8C%AE%E8%80%85%E6%B5%B7%E6%8A%A5.png)

## Highlights of Recent Features

### Apache APISIX

- [APISIX allows customizing prometheus default bucket](https://github.com/apache/apisix/pull/9673) (Contributor: [jiangfucheng](https://github.com/jiangfucheng))

- [Fix the problem that body-transformer plugin cannot convert empty tables properly](https://github.com/apache/apisix/pull/9669) (Contributor: [kingluo](https://github.com/kingluo))

- [Fix the issue that max_kept configuration of the log-rotate plugin does not work when using a custom name](https://github.com/apache/apisix/pull/9749) (Contributor: [monkeyDluffy6017](https://github.com/monkeyDluffy6017))

> ### Apache APISIX Ingress

Reviewer: ```suggestion
### Apache APISIX Ingress Controller
```

---
title: "Biweekly Report (June 19 - July 05)"
keywords: 
- Apache APISIX
- API Gateway
- Weekly Report
- Contributor
description: The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
tags: [Community]
image: https://static.apiseven.com/uploads/2023/07/03/o1PFlwn0_%E7%A4%BE%E5%8C%BA%E5%8F%8C%E5%91%A8%E6%8A%A5-%E5%85%AC%E4%BC%97%E5%8F%B7%E5%A4%B4%E5%9B%BE-%E8%8B%B1%E6%96%87.png
---

> > We have recently fixed and improved some features of Apache APISIX and Apache APISIX Ingress Controller, including optimizing Prometheus, fixing the body-transformer and log-rotate plugins, and adding an annotation to allow rewriting of the response header. For more details, please read this bi-weekly report. Meanwhile, We are pleased to present Apache APISIX 3.4.0 with exciting new features and performance improvements.

Reviewer: ```suggestion
> We have recently fixed and improved some features of Apache APISIX, including optimizing the use of Prometheus, fixing the body-transformer and log-rotate plugins, and adding an annotation to allow rewriting of the response header for Apache APISIX Ingress Controller. Meanwhile, We are pleased to present Apache APISIX 3.4.0 with exciting new features and performance improvements. For more details, please read this biweekly report.
```

## Contributor Statistics

![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2023/07/03/NmhS4JKj_%E8%B4%A1%E7%8C%AE%E8%80%85%E5%90%8D%E5%8D%95.png)

![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2023/07/07/M3P3D5AP_%E6%96%B0%E6%99%8B%E8%B4%A1%E7%8C%AE%E8%80%85%E6%B5%B7%E6%8A%A5.png)

## Highlights of Recent Features

### Apache APISIX

- [APISIX allows customizing prometheus default bucket](https://github.com/apache/apisix/pull/9673) (Contributor: [jiangfucheng](https://github.com/jiangfucheng))

- [Fix the problem that body-transformer plugin cannot convert empty tables properly](https://github.com/apache/apisix/pull/9669) (Contributor: [kingluo](https://github.com/kingluo))

- [Fix the issue that max_kept configuration of the log-rotate plugin does not work when using a custom name](https://github.com/apache/apisix/pull/9749) (Contributor: [monkeyDluffy6017](https://github.com/monkeyDluffy6017))

### Apache APISIX Ingress Controller

> - [APISIX Ingress adds annotation to allow rewriting of response headers](https://github.com/apache/apisix-ingress-controller/pull/1861) (Contributor: [Revolyssup](https://github.com/Revolyssup))

Reviewer: ```suggestion
- [APISIX Ingress Controller adds annotation to allow rewriting of response headers](https://github.com/apache/apisix-ingress-controller/pull/1861) (Contributor: [Revolyssup](https://github.com/Revolyssup))
```
